src/blockstore/store.ts

```diff
@@ -465,3 +508,48 @@ export class WALStoreImpl extends BaseStoreImpl implements WALStore {
     return this.gateway.destroy(this.url());
   }
 }
+
```

Comment:
I had trouble with the node module in the smoke test

Reply:
https://github.com/fireproof-storage/fireproof/actions/runs/10751859938/job/29819554791